test/views/toolbar_view_test.exs

```diff
@@ -7,111 +7,123 @@ defmodule ExDebugToolbar.ToolbarViewTest do
-    assert request |> render |> is_bitstring
-  end
+    test "it renders toolbar with ecto query with nil queue and decode times" do
+      log_entry = %Ecto.LogEntry{
+        decode_time: nil,
+        query: "select * from users",
+        query_time: 1550583,
+        queue_time: nil,
+        source: "users",
+        result: {:ok, %Postgrex.Result{}}
+      }
+      duration = 15000
+      request = %Request{ecto: [{log_entry, duration, :inline}]}
+      assert request |> render |> is_bitstring
+    end
 
-  test "it renders toolbar with timeline" do
-    timeline = %Timeline{
-      duration: 50,
-      events: [%Timeline.Event{
-        name: "controller.call",
-        duration: 5,
+    test "it renders toolbar with timeline" do
+      timeline = %Timeline{
+        duration: 50,
         events: [%Timeline.Event{
-          name: "controller.render",
-          duration: 7,
+          name: "controller.call",
+          duration: 5,
           events: [%Timeline.Event{
-            name: "template#app.html",
-            duration: 10
+            name: "controller.render",
+            duration: 7,
+            events: [%Timeline.Event{
+              name: "template#app.html",
+              duration: 10
+            }]
           }]
         }]
-      }]
-    }
-    request = %Request{timeline: timeline}
-    assert request |> render |> is_bitstring
-  end
+      }
+      request = %Request{timeline: timeline}
+      assert request |> render |> is_bitstring
+    end
 
-  test "it renders toolbar with timeline that has controller.call event only" do
-    timeline = %Timeline{
-      duration: 50,
-      events: [%Timeline.Event{
-        name: "controller.call",
-        duration: 5,
-        }]
-    }
-    request = %Request{timeline: timeline}
-    assert request |> render |> is_bitstring
-  end
+    test "it renders toolbar with timeline that has controller.call event only" do
+      timeline = %Timeline{
+        duration: 50,
+        events: [%Timeline.Event{
+          name: "controller.call",
+          duration: 5,
+          }]
+      }
+      request = %Request{timeline: timeline}
+      assert request |> render |> is_bitstring
+    end
 
-  test "it renders toolbar with breakpoints" do
-    breakpoint = %Breakpoint{
-      id: 1,
-      pid: self(),
-      line: 5,
-      file: "test.ex",
-      code_snippet: [{"a = [1, 2]", 5}],
-      env: __ENV__,
-      binding: binding(),
-      inserted_at: NaiveDateTime.utc_now
-    }
-    assert %Request{} |> render(breakpoints: [breakpoint]) |> is_bitstring
-  end
+    test "it renders toolbar with breakpoints" do
+      breakpoint = %Breakpoint{
+        id: 1,
+        pid: self(),
+        line: 5,
+        file: "test.ex",
+        code_snippet: [{"a = [1, 2]", 5}],
+        env: __ENV__,
+        binding: binding(),
+        inserted_at: NaiveDateTime.utc_now
+      }
+      assert %Request{} |> render(breakpoints: [breakpoint]) |> is_bitstring
+    end
```

Comment:
I think we need a similar test that renders with multiple requests in history to ensure panel renders without errors